src/msg/async/ProtocolV1.cc

```diff
@@ -341,19 +341,27 @@ void ProtocolV1::write_event() {
         prepare_send_message(connection->get_features(), m, data);
       }
 
-      r = write_message(m, data, more);
+      write_message(m, data, more);
```

Comment:
no need to pass `more` to `write_message()` anymore. and might want to make `write_message()` return `void` .